### ISSUE TITLE
Should we provide negative samples?

### ISSUE THREAD

**Issue by HuangBo-Terraloupe** [yhenon/keras-frcnn#182]

**Comment by HuangBo-Terraloupe** [comment]
My dataset contains a lot of negative samples, a lot of images do not contain any object.
After the training, I save the model with best validation loss, but the results looks really bad.
On the test dataset, faster rcnn does not generator bounding box for a lot of test image patches.

![029982](https://user-images.githubusercontent.com/26878250/31010008-a856deee-a509-11e7-8bce-bb9470040a9e.jpg)
